sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipRoutingClient.cs

```diff
@@ -130,18 +130,84 @@ public virtual Response<SipTrunk> GetTrunk(
+        /// <returns>Trunk configuration.</returns>
+        /// <exception cref="KeyNotFoundException">Route with specified name wasn't found.</exception>
+        public virtual Response<SipTrunk> GetTrunk(
+            string fqdn,
+            ExpandEnum expand,
+            CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(SipRoutingClient)}.{nameof(GetTrunk)}");
+            scope.Start();
+            try
+            {
+                var response = _restClient.Get(expand, cancellationToken);
+                var trunk = response.Value.Trunks[fqdn];
+
+                if (trunk == null)
+                {
+                    throw new KeyNotFoundException($"SIP trunk with FQDN: {fqdn} wasn't found");
+                }
+                return Response.FromValue(trunk, response.GetRawResponse());
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get <see cref="SipTrunk"/> with provided FQDN.
+        /// </summary>
+        /// <param name="fqdn">SIP trunk FQDN.</param>
+        /// <param name="cancellationToken">Optional cancellation token.</param>
+        /// <returns>Trunk configuration.</returns>
+        /// <exception cref="KeyNotFoundException">Route with specified name wasn't found.</exception>
+        public async virtual Task<Response<SipTrunk>> GetTrunkAsync(
+            string fqdn,
+            CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(SipRoutingClient)}.{nameof(GetTrunk)}");
+            scope.Start();
+            try
+            {
+                var response = await _restClient.GetAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
+                var trunk = response.Value.Trunks[fqdn];
+
+                if (trunk == null)
+                {
+                    throw new KeyNotFoundException($"SIP trunk with FQDN: {fqdn} wasn't found");
+                }
+                return Response.FromValue(trunk, response.GetRawResponse());
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get <see cref="SipTrunk"/> with provided FQDN.
+        /// </summary>
+        /// <param name="fqdn">SIP trunk FQDN.</param>
+        /// <param name="expand">Expand enum.</param>
         /// <param name="cancellationToken">Optional cancellation token.</param>
         /// <returns>Trunk configuration.</returns>
         /// <exception cref="KeyNotFoundException">Route with specified name wasn't found.</exception>
         public async virtual Task<Response<SipTrunk>> GetTrunkAsync(
             string fqdn,
+            ExpandEnum expand,
```

Comment:
Feedback from review:
don't use ExpandEnum in public methods, let's use:
 boolean includeHealth parameter instead.